testing/MLDB-1336-builtin-checks.py

```diff
@@ -0,0 +1,21 @@
+
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+import unittest
+
+class myTest(unittest.TestCase):
+
+    def test_sequence(self):
+        with self.assertRaisesRegexp(mldb_wrapper.ResponseException, "Executing builtin function exp: Can't convert value 'a' to double") as re:
```

Comment:
Because its at execution, we wont know the problem is with an argument unless we put a try-catch within every builtin.